.npmrc

```diff
@@ -0,0 +1,3 @@
+strict-peer-dependencies = false
+auto-install-peers = true
+enable-pre-post-scripts = true
```

Comment:
I don't have a very strong opinion about this but I like the [arguments](https://github.com/pnpm/pnpm/issues/2891) against setting this option. I'm fine with keeping it as is. Just wanted to share so we can keep it in mind.

Reply:
```
strict-peer-dependencies = false
auto-install-peers = true
```
had some issues with dependencies in the exact same setup (next + hardhat + pnpm) previously that made me to add those lines. but can't recall. i'll try without.

```
enable-pre-post-scripts = true
```
we needed this before turborepo to compile contracts before starting the dev environment. but good catch, now we don't need it anymore.

Reply:
yeah i remembered correctly (see output below). if you're fine i would keep the first two lines, i always liked the behavior of automatically having peers installed more... 

but let me know if you have a strong opinion about it.

```
 ERR_PNPM_PEER_DEP_ISSUES  Unmet peer dependencies

.
├─┬ eslint-config-custom
│ ├─┬ eslint-config-next
│ │ ├── ✕ missing peer eslint@"^7.23.0 || ^8.0.0"
│ │ ├─┬ @typescript-eslint/parser
│ │ │ └── ✕ missing peer eslint@"^6.0.0 || ^7.0.0 || ^8.0.0"
│ │ ├─┬ eslint-import-resolver-typescript
│ │ │ └── ✕ missing peer eslint@"*"
│ │ ├─┬ eslint-plugin-import
│ │ │ └── ✕ missing peer eslint@"^2 || ^3 || ^4 || ^5 || ^6 || ^7.2.0 || ^8"
│ │ ├─┬ eslint-plugin-jsx-a11y
│ │ │ └── ✕ missing peer eslint@"^3 || ^4 || ^5 || ^6 || ^7 || ^8"
│ │ ├─┬ eslint-plugin-react
│ │ │ └── ✕ missing peer eslint@"^3 || ^4 || ^5 || ^6 || ^7 || ^8"
│ │ └─┬ eslint-plugin-react-hooks
│ │   └── ✕ missing peer eslint@"^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0-0"
│ ├─┬ eslint-config-prettier
│ │ └── ✕ missing peer eslint@>=7.0.0
│ └─┬ eslint-plugin-react
│   └── ✕ missing peer eslint@"^3 || ^4 || ^5 || ^6 || ^7 || ^8"
└─┬ next
  ├── ✕ missing peer react@"^17.0.2 || ^18.0.0-0"
  ├── ✕ missing peer react-dom@"^17.0.2 || ^18.0.0-0"
  ├─┬ styled-jsx
  │ └── ✕ missing peer react@">= 16.8.0 || 17.x.x || ^18.0.0-0 || ^17.0.2"
  └─┬ use-sync-external-store
    └── ✕ missing peer react@"^16.8.0 || ^17.0.0 || ^18.0.0 || ^17.0.2"
Peer dependencies that should be installed:
  eslint@">=7.23.0 <8.0.0 || >=8.0.0-0 <8.0.0 || >=8.0.0 <9.0.0"      react-dom@"^17.0.2 || ^18.0.0-0"                                    react@">=17.0.2 <18.0.0 || >=18.0.0-0 <18.0.0 || >=18.0.0 <19.0.0"  

packages/frontend
├─┬ babel-jest
│ ├── ✕ missing peer @babel/core@^7.8.0
│ └─┬ babel-preset-jest
│   ├── ✕ missing peer @babel/core@^7.0.0
│   └─┬ babel-preset-current-node-syntax
│     ├── ✕ missing peer @babel/core@^7.0.0
│     ├─┬ @babel/plugin-syntax-async-generators
│     │ └── ✕ missing peer @babel/core@^7.0.0-0
│     ├─┬ @babel/plugin-syntax-bigint
│     │ └── ✕ missing peer @babel/core@^7.0.0-0
│     ├─┬ @babel/plugin-syntax-class-properties
│     │ └── ✕ missing peer @babel/core@^7.0.0-0
│     ├─┬ @babel/plugin-syntax-import-meta
│     │ └── ✕ missing peer @babel/core@^7.0.0-0
│     ├─┬ @babel/plugin-syntax-json-strings
│     │ └── ✕ missing peer @babel/core@^7.0.0-0
│     ├─┬ @babel/plugin-syntax-logical-assignment-operators
│     │ └── ✕ missing peer @babel/core@^7.0.0-0
│     ├─┬ @babel/plugin-syntax-nullish-coalescing-operator
│     │ └── ✕ missing peer @babel/core@^7.0.0-0
│     ├─┬ @babel/plugin-syntax-numeric-separator
│     │ └── ✕ missing peer @babel/core@^7.0.0-0
│     ├─┬ @babel/plugin-syntax-object-rest-spread
│     │ └── ✕ missing peer @babel/core@^7.0.0-0
│     ├─┬ @babel/plugin-syntax-optional-catch-binding
│     │ └── ✕ missing peer @babel/core@^7.0.0-0
│     ├─┬ @babel/plugin-syntax-optional-chaining
│     │ └── ✕ missing peer @babel/core@^7.0.0-0
│     └─┬ @babel/plugin-syntax-top-level-await
│       └── ✕ missing peer @babel/core@^7.0.0-0
├─┬ @emotion/react
│ ├── ✕ missing peer @babel/core@^7.0.0
│ └─┬ @emotion/babel-plugin
│   ├── ✕ missing peer @babel/core@^7.0.0
│   └─┬ @babel/plugin-syntax-jsx
│     └── ✕ missing peer @babel/core@^7.0.0-0
├─┬ @emotion/styled
│ └── ✕ missing peer @babel/core@^7.0.0
├─┬ next
│ └─┬ styled-jsx
│   └── ✕ missing peer @babel/core@"*"
└─┬ wagmi
  └─┬ @coinbase/wallet-sdk
    └─┬ eth-block-tracker
      └─┬ @babel/plugin-transform-runtime
        ├── ✕ missing peer @babel/core@^7.0.0-0
        ├─┬ babel-plugin-polyfill-corejs2
        │ ├── ✕ missing peer @babel/core@^7.0.0-0
        │ └─┬ @babel/helper-define-polyfill-provider
        │   ├── ✕ missing peer @babel/core@^7.4.0-0
        │   └─┬ @babel/helper-compilation-targets
        │     └── ✕ missing peer @babel/core@^7.0.0
        ├─┬ babel-plugin-polyfill-corejs3
        │ └── ✕ missing peer @babel/core@^7.0.0-0
        └─┬ babel-plugin-polyfill-regenerator
          └── ✕ missing peer @babel/core@^7.0.0-0
Peer dependencies that should be installed:
  @babel/core@">=7.8.0 <8.0.0"  

packages/hardhat
└─┬ @typechain/ethers-v5
  └── ✕ missing peer @ethersproject/bytes@^5.0.0
Peer dependencies that should be installed:
  @ethersproject/bytes@^5.0.0  

hint: If you want peer dependencies to be automatically installed, add "auto-install-peers=true" to an .npmrc file at the root of your project.
hint: If you don't want pnpm to fail on peer dependency issues, add "strict-peer-dependencies=false" to an .npmrc file at the root of your project.
```

Reply:
update: actually i would like to keep all three lines as esp. the third matches better with the default npm-behavior and somehow also makes more sense imo.